Kickstarter-iOS/Views/CommentTableViewFooter.swift

```diff
@@ -0,0 +1,43 @@
+import Library
+import Prelude
+import UIKit
+
+final class CommentTableViewFooter: UIView {
+  // MARK: - Properties
+
+  private lazy var activityIndicator: UIActivityIndicatorView = {
+    let indicator = UIActivityIndicatorView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+    indicator.hidesWhenStopped = true
+    return indicator
+  }()
+
+  public var shouldShowActivityIndicator: Bool = false {
```

Comment:
Not sure if this should be updated via a view model? cc @justinswart

Reply:
We almost always use a view model output to update a component of the UI. e.g `.hidden`